Jenkinsfile_CNP

```diff
@@ -12,100 +12,100 @@ def dmnBranch = "master"
 def waStandaloneBranch = "master"
 
 def secrets = [
-    'sscs-${env}': [
```

Comment:
All these look like just a whitespace change - can you confirm and if not needed, revert to master please